<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCustomers</class>
 <widget class="QDialog" name="EditCustomers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 170, 0)</string>
  </property>
  <widget class="QPushButton" name="buttonModifyCustomer">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>176</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modify customer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastname">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>350</y>
     <width>133</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background: white</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstname">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>410</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Firstname</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLastname">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>108</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lastname</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelectTheFunction">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>214</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select the function</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomerInformation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>254</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer information</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomerID">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>143</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreateNewCustomer">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>280</y>
     <width>214</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create new customer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>700</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstname">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>410</y>
     <width>133</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerID">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>281</y>
     <width>133</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInfo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>510</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
